gameofgame.py

```diff
@@ -20,39 +20,50 @@
 def choose_country():
     country = random.choice(list(countries_capitals.keys()))
     return country
+    
 
 # Function to check if the guess is correct
 def check_guess(guess, country):
     capital = countries_capitals[country]
     if guess.lower() == capital.lower():
         return True
     else:
-        return False
+       return False
```

Comment:
Looks like there is inconsistent indentation here.

Reply:
OK, so now you have the same indentation on both these lines, but it is inconsistent with the rest of the file. All indentation in a Python module needs to be a multiple of 4 spaces.